build/teamcity-release-notes.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+cockroach_ref=$(git describe --tags --exact-match 2>/dev/null || git rev-parse HEAD)
+
+# A merge may contain multiple PRs due to craig(bot) activity
+# Get PR numbers for current merge
+curl -H 'Authorization: token $GITHUB_API_TOKEN' https://api.github.com/search/issues?q=sha:$cockroach_ref+repo:cockroachdb/cockroach | jq .items[].number > prnums
+prcount=$(cat prnums | wc -l)
+
+# For each PR, process its commit(s)
+process_pr_commits() {
+  # Get commits given PR number
+  commits=$(curl -H 'Authorization: token $GITHUB_API_TOKEN' https://api.github.com/repos/cockroachdb/cockroach/pulls/$1/commits)
+  commits="${commits//$'\''/'%27'}"
+  echo $commits > commits.json
```

Comment:
nit: not really sure you need `commits.json`, you can just pipe `$commits` into `jq` wherever you need it.

Reply:
+1 to usually using variables instead of files.

If files are used anywhere, I'd ask that they be cleaned up via a `trap` function call. [Here](https://github.com/cockroachdb/cockroach/blob/6651a08752080a3594714b9e8de009bf4155fbb1/build/teamcity-support.sh#L8-L12) is an example of how that works.

Reply:
If you don't mind, I would prefer to keep the temporary variables and files in place for ease of debugging, code readability, and ease of future maintenance. I'll work on cleaning up via `trap` as suggested.

Reply:
Added the clean-up. Thank you both.